Makefile

```
@@ -67,7 +67,7 @@ ifeq (,$(shell which heighliner))
 	@echo heighliner not found. https://github.com/strangelove-ventures/heighliner
 else
 	@echo "🤖 Building image..."
-	@heighliner build --chain noble-authority-simd --local 1> /dev/null
+	@heighliner build --file ./e2e/chains.yaml --chain noble-authority-simd --local
```

This might just be me, but I quite like silencing the output of Heighliner, as it gets quite noisy.

```suggestion
	@heighliner build --file ./e2e/chains.yaml --chain noble-authority-simd --local 1> /dev/null
```

---

This makes troubleshooting Heighliner difficult, which I had to do in this PR. 
Because building images with Heighliner is targeted for such a specific user, I think it is OK to keep output here.
Happy to redirect if you disagree!